### DISCUSSION TITLE
what is the 'Fallback to older releases' option in app?

### DISCUSSION THREAD

**Discussion by soymadip** [ImranR98/Obtainium#585] [Q&A] [answered]
What does it do?

**Comment by DwainZwerg** [comment, marked as answer]
The "Fallback to older releases" toggle is there to let Obtainium fall back to an older version of the app if the latest one does not support the processor built into the smartphone.
Example: The latest version of Google Play Services for AR is 1.37.231000483, but this is only available for armv8a. The older version 1.37.231000423 is also available for armabiv7a.

**Reply by ImranR98** [reply]
Yep, if the latest release doesn't have an APK you want (after applying any APK filters you may have set), it checks older releases.

**Reply by soymadip** [reply]
Ohh i get it
